prime-router/metadata/fhir_mapping/hl7/segments/MSH/datatypes/HD/Destination.yml

```diff
@@ -0,0 +1,87 @@
+name:
+  type: STRING
+  expressionType: HL7Spec
+  valueOf: HD.1
+
+endpoint:
+  vars:
+    hd2: STRING, HD.2
+    hd3: STRING, HD.3
+  constants:
+    oidUrlPrefix: "urn:oid:"
+    uuidUrlPrefix: "urn:uuid:"
+  expressionType: nested
+  expressions:
+    - condition: $hd2 NOT_NULL && $hd3 EQUALS ISO
+      vars:
+        endpointValue: $oidUrlPrefix + $hd2
+      type: STRING
+      valueOf: $endpointValue
+    - condition: $hd2 NOT_NULL && $hd3 EQUALS UUID
+      vars:
+        endpointValue: $uuidUrlPrefix + $hd2
+      type: STRING
+      valueOf: $endpointValue
```

Comment:
I vaguely remember us talking about accounting for DNS and CLIA in these even if the mapping doesn't reference those values...do you remember that?

Reply:
Vaguely...though now that I've seen how inconsistent all that is my take is that we match the inventory and then figure out a consistent internal opinion after